projects/fsastorefrontstyles/scss/base/building-blocks.scss

```diff
@@ -240,11 +250,6 @@
 .opaque {
   opacity: 0.5;
 }
-.sticky-position {
```

Comment:
Removed this class and use our classes `top` and `z-index-1` with Bootstrap class `position-sticky` wherever we had assigned `sticky-position` class.